README.md

```diff
@@ -26,7 +27,7 @@ fn main() {
     let iter = vec![0, 1, 2, 3, 4, 5, 6, 7, 8, 9].into_iter();
     let stream = stream::iter_ok::<_, io::Error>(iter);
 
-    let chunk_stream = Chunks::new(stream, 5, Duration::new(10, 0));
+    let chunk_stream = ChunksTimeout::new(stream.compat(), 5, Duration::new(10, 0));
```

Comment:
Shall we mention here that `compat()` is needed to run  `std` futures on executors built for `futures@0.1`?
Maybe with a link to this document:
https://rust-lang-nursery.github.io/futures-rs/blog/2019/04/18/compatibility-layer.html

Reply:
Yes. However note that this example right now doesn't compile, as I haven't figured out the issue with `skeptic` yet.